lti_consumer/lti_xblock.py

```diff
@@ -131,6 +132,26 @@ def parse_handler_suffix(suffix):
     raise LtiError(msg)
 
 
+def config_type_values_provider(block):
+    """
+    Return a list of possible values for the config_type xBlock field.
```

Comment:
maybe call this allowed_config_types(block) or possible_config_types(block) that obviously returns a set of types rather than using the jargony "provider" which might return some weird model?